Example/App/Resources/Images.xcassets/Contents.json

```
@@ -0,0 +1,6 @@
+{
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
+}
```

A new line at file end should be added

```suggestion
}

```